### PR TITLE
Add 1 20 announcement

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2020-10-26-announcing-fusionauth-1-20.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Finally FusionAuth now honors the `AssertionConsumerServiceURL` in a SAML request. Multiple redirect URLs can be configured in the application SAML settings as well.
+
+### Docker changes
+
+In this release, we updated the base image for Docker from `alpine` to `ubuntu:focal`. This should not impact functionality, but it's worth paying attention to if you are building Docker images based on our image.
+
+We made this change because in order to run on `alpine` without including the GNU C Library (`glibc`) we had to use a custom build of OpenJDK compiled using the `musl` C library. Due to some possible performance concerns with this option, we have moved to an official build of JDK provided by AdoptOpenJDK. This build is compiled using `glibc`. Using `ubuntu:focal` increased the image size by approximately 30 MB compared to the `alpine` image, unfortunately. However, until we can obtain official builds from AdoptOpenJDK based upon the `musl` C library, we are not planning to ship an official FusionAuth image on `alpine` due to the performance concerns.
```
Comment (dpolstra) [inline site/_posts/2020-10-26-announcing-fusionauth-1-20.md:33]: Unfortunately, using `ubuntu:focal` increased the image size by approximately 30 MB compared to the `alpine` image.

File: site/_posts/2020-10-26-announcing-fusionauth-1-20.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Additional admin forms
```
Comment (dpolstra) [inline site/_posts/2020-10-26-announcing-fusionauth-1-20.md:35]: Additional admin registration forms

**Comment by voidmain** [comment]
I added a few tweaks directly to the code. Looks good!

File: site/_posts/2020-10-26-announcing-fusionauth-1-20.md (hunk 2 above)
Comment (mooreds) [reply site/_posts/2020-10-26-announcing-fusionauth-1-20.md:35]: They aren't just for registration, though. How about "Additional admin user management forms"?

**Pull request merged**